### DISCUSSION TITLE
Send E-Mail after submitting form

### DISCUSSION THREAD

**Discussion by ben-biber** [Budibase/budibase#10596] [Help] [answered]
I want to create a form that's does the following:

1. Public User fills out the form
2. Public User submits form
3. Public User receives e-mail with the data he submitted

Step 1. and 2 were easy to accomplish but Im stuck at step 3.
Does someone has a guide or could explain me how I would do this?

**Comment by shogunpurple** [comment, marked as answer]
https://docs.budibase.com/docs/email
